test/jest/fixtures.js

```diff
@@ -131,12 +137,76 @@ module.exports = function registerFixtures(global) {
     await test(playwright.toImpl);
   });
 
+  /** @type {Set<Set<Browser|BrowserServer|BrowserContext>>} */
+  const allClosables = new Set();
+
+  process.on('SIGINT', () => {
+    const promises = [];
+    for (const closables of allClosables) {
+      for (const closable of closables)
+        promises.push(closable.close());
+    }
+    Promise.all(promises).then(() => process.exit(130));
+  });
+
   global.registerWorkerFixture('browserType', async ({playwright}, test) => {
-    await test(playwright[process.env.BROWSER || 'chromium']);
+    /** @type {BrowserType} */
+    const browserType = playwright[process.env.BROWSER || 'chromium'];
+    /** @type {Set<Browser|BrowserServer|BrowserContext>} */
+    const closables = new Set();
+    /** @type {BrowserType} */
+    const wrapper = {
+      ...browserType,
+      async connect(options) {
```

Comment:
We no longer test browserType and this almost doubles our fixtures file. Let's spend some time brainstorming this.

Reply:
Can I say `this.global.process = process` ?

Reply:
This calls every method on browserType in the same way as before. I don't understand.

Reply:
Wrapping the browser type inline suggests that we are doing something wrong. If we believe this is a way to go, we should implement a prominent class that has a graceful close method. Alternatively, it could be a public method on the browser type itself.

Reply:
Our api has support for one browser being closed via sigint with handleSIGINT: true. If you have multiple browsers, we suggest handleSIGINT false, tracking all of them, and closing them yourself. We can make this the default behavior of `browserType`, however I'd still like to land this and make sure it works well before committing to it as the public behavior.